src/app/components/client/LocationAutocomplete.module.scss

```diff
@@ -0,0 +1,16 @@
+@import "../../tokens";
+
+.locationItem {
+  font: $text-body-sm;
+  padding: $spacing-sm calc($spacing-sm * 1.5);
+
+  strong {
+    letter-spacing: 0.01em;
+  }
```

Comment:
question: This rule feels really specific. What was the issue here? Is this browser-specific?

Reply:
Added in https://github.com/mozilla/blurts-server/pull/3217/commits/1b195d87775e45b003abe06b666fbd4b4a4f0aea.